models/classes/tasks/QtiStateOffload/AbstractQtiStateManipulationTask.php

```diff
@@ -27,26 +27,35 @@
 use oat\oatbox\reporting\Report;
 use oat\oatbox\service\exception\InvalidServiceManagerException;
 use oat\tao\model\state\StateMigration;
+use oat\tao\model\taskQueue\Task\QueueAssociableInterface;
 use oat\tao\model\taskQueue\Task\TaskAwareInterface;
 use oat\tao\model\taskQueue\Task\TaskAwareTrait;
 use Psr\Container\ContainerExceptionInterface;
 use Psr\Container\NotFoundExceptionInterface;
 
-abstract class AbstractQtiStateManipulationTask extends AbstractAction implements TaskAwareInterface
+abstract class AbstractQtiStateManipulationTask extends AbstractAction
+    implements TaskAwareInterface, QueueAssociableInterface
```

Comment:
`QueueAssociableInterface` does the job here. But in case if we need to change it back, we have to update the code again.
WDYT about using taskqueue config to associate tasks to their queue? the result will be the same, but need to specifically reference targetted tasks.

Reply:
i like that approach also because it's allow to define queue name once exactly in related abstract class which used as base. But agree, that your proposition is more flexible, will apply it